account_statement_import_online_gocardless/models/online_bank_statement_provider.py

```diff
@@ -190,6 +190,39 @@ def action_check_gocardless_agreement(self):
             # JS code expects here to return a plain link or nothing
             return requisition_data["link"]
 
+    def _gocardless_request_requisition(self):
+        response = requests.get(
+            f"{GOCARDLESS_ENDPOINT}/requisitions/{self.gocardless_requisition_id}/",
+            headers=self._gocardless_get_headers(),
+            timeout=REQUESTS_TIMEOUT,
+        )
+        if response.status_code == 200:
+            return json.loads(response.text)
+        else:
+            return {}
```

Comment:
And the same for the rest.

```suggestion
        if response.status_code == 200:
            return json.loads(response.text)
        return {}
```

Reply:
Done :) thank you for your time!